bower.json

```diff
@@ -18,6 +18,8 @@
     "angular": "~1.4.5",
     "angular-animate": "~1.4.5",
     "angular-route": "~1.4.5",
-    "bootstrap": "~3.3.5"
+    "bootstrap": "~3.3.5",
+    "raphael": "raphael.js#~2.1.4",
+    "sigma.js": "*"
```

Comment:
Weird!  I just did `bower install sigma.js --save`.  Anyway, I'm not using sigma now so I'll just remove this.